android/src/app/organicmaps/editor/EditorHostFragment.java

```diff
@@ -301,16 +301,8 @@ public void onClick(View v)
       {
       case OPENING_HOURS:
         final String timetables = ((TimetableContainerFragment) getChildFragmentManager().findFragmentByTag(TimetableContainerFragment.class.getName())).getTimetable();
-        if (OpeningHours.nativeIsTimetableStringValid(timetables))
-        {
-          Editor.nativeSetOpeningHours(timetables);
-          editMapObject();
-        }
-        else
-        {
-          // TODO (yunikkk) correct translation
-          showMistakeDialog(R.string.editor_correct_mistake);
-        }
+        Editor.nativeSetOpeningHours(timetables);
```

Comment:
This change should be squashed into the first commit.

Reply:
Squashed.